src/components/hero/Hero.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="hero">
+    <div class="content">
+      <h1>More than just shorter links</h1>
```

Comment:
I think `H1` should be `Shortly` in navigation component. Maybe `h2` would be better choose.